Source/Core/DolphinQt2/Config/Graphics/GraphicsWidget.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Config/Graphics/GraphicsWidget.h"
+
+#include <QEvent>
+#include <QFormLayout>
+#include <QGroupBox>
+#include <QLabel>
+#include <QVBoxLayout>
+
+#include "DolphinQt2/Config/Graphics/GraphicsWindow.h"
+
+GraphicsWidget::GraphicsWidget(GraphicsWindow* parent)
+{
+  parent->RegisterWidget(this);
+}
+
+void GraphicsWidget::AddDescription(QWidget* widget, const char* description)
```

Comment:
Unneeded as `DescriptionAdded` can be called directly (`emit` is a blank macro).